tensorrt_llm/_torch/models/modeling_hunyuan_dense.py

```
@@ -0,0 +1,602 @@
+
+                                if isinstance(out, torch.Tensor):
+                                    shape = list(out.shape)
+                                    output_values, output_mid_values, output_back_values, output_mean, output_var = _get_tensor_stats(
+                                        out)
+                                elif isinstance(
+                                        out,
+                                    (tuple,
+                                     list)) and len(out) > 0 and isinstance(
+                                         out[0], torch.Tensor):
+                                    shape = list(out[0].shape)
+                                    output_values, output_mid_values, output_back_values, output_mean, output_var = _get_tensor_stats(
```

Ruff (0.12.2)

Line too long (134 > 120)

(E501)